lib/ramble/ramble/config.py

```diff
@@ -84,9 +87,12 @@
     "modifier_repos": ramble.schema.modifier_repos.schema,
     "modifiers": ramble.schema.modifiers.schema,
     "spack": ramble.schema.spack.schema,
+    "software": ramble.schema.software.schema,
     "success_criteria": ramble.schema.success_criteria.schema,
+    "package_manager_repos": ramble.schema.package_manager_repos.schema,
     "applications": ramble.schema.applications.schema,
     "variables": ramble.schema.variables.schema,
+    "variants": ramble.schema.variants.schema,
```

Comment:
Would it make sense to make `ramble:variants:package_manager` to default to `spack` instead of none, given that that was the de facto existing behavior?

Reply:
I see, makes sense.